todo_app/app.py

```diff
@@ -1,5 +1,7 @@
-from flask import Flask
+#import todo_app.data.session_items as module_SI
```

Comment:
Nice - this is a totally valid alternative and it's worth exploring both; there are different opinions for when to prefer each but generally as long as you're avoiding importing everything blindly (e.g. `from <module> import *`) it's largely down to preference

That said, once we're happy with our set up I'm a big proponent of removing old commented out code - it quickly becomes cruft that adds overhead for working with a code base, which we could always recover from Git anyway if we need to